### ISSUE TITLE
Add legendas.tv bigger PT-BR Subtitle Provider

### ISSUE THREAD

**Issue by felipesalomao** [Diaoul/subliminal#383]
Hi @Diaoul, you told that would be better develop add providers when release subliminal 0.7, and we are on 7.0.4 now :) 
Legendas.tv was totally remodeled a few months ago and does not require authentication anymore. In fact, many things changed and it's now much simpler to make the provider since the URLs are straight links and not a mess of javascript and redirections like before. Bigger problem told by @Diaoul before, was fact that would be necessary user/pass to download subtitles from legendas.tv provider, now does not need anymore.. Can download any subtitle without need user and pass.. Hope develop it..
i think that could help: https://github.com/MestreLion/legendastv

Thank you for great work !

**Comment by ratoaq2** [comment]
I have started some experimental work on this. Subliminal architecture is awesome, I must say.

Legendas.tv has a really poor API/interface. The data is not exposed in a structured way and it makes a little bit harder than other providers. Other providers are much easier since they either use hashes or known ids (imdb_id or tvdb_id). Legendas.tv does not even distinguish between movies and shows and they have a lot of duplicated entries (duplicated movie entries, duplicated shows entries). 

One thing that I noticed: Subliminal scoring system does not have a field related to the "subtitle quality". Is this by design?
If I have several subtitles that matches exactly my release, then we could use some information from the provider (number of downloads, user rating, or any other field) to choose the one with best quality.

Just to give an example:
```
Subtitle 1:
Show A S01E01 720p HDTV DIMENSION.srt
Rate: 3 stars, 5 downloads
```

```
Subtitle 2:
Show A S01E01 720p HDTV DIMENSION.srt
Rate: 9 stars, 230192 downloads
```

Their score will be the same, but I would rather prefer to get Subtitle 2

**Comment by Diaoul** [comment]
I will add this feature for 1.2 and update the providers accordingly.

**Comment by felipesalomao** [comment]
Hi, your work to add legendas.tv provider is awesome!!! 
It is full developed (seems To be) ? 

It already was added and is available to any one ? I think not because o cant see legendastv in providers files and also actual version is 1.0.1, You said that it would be added in 1.2 version, if Yes, do You know when You will release it ?

Thanks for Great work !

**Comment by Diaoul** [comment]
Fixed with 9785bc9746c24c8c28c4ed20812f103a070ca357
